README.md

```diff
@@ -64,37 +56,29 @@ ZCLI supports numerous commands. Further documentation on available commands can
 - [`$ zcli autocomplete`](/docs/autocomplete.md) - display autocomplete installation instructions.
 - [`$ zcli help`](/docs/help.md) - display help for zcli
 
-# Developing
+# Contribution
```

Comment:
Maybe instead of the passive `Contribution` form of the verb, perhaps consider the active tense: `Contributing`